dts/arm/renesas/ra/ra-cm4-common.dtsi

```diff
@@ -82,19 +85,64 @@
 			#interrupt-cells = <3>;
 		};
 
-		cgc: cgc@4001e000 {
-			compatible = "renesas,ra-clock-generation-circuit";
-			reg = <0x4001e000 0x40 0x40047000 0x10>;
-			reg-names = "system", "mstp";
-			#clock-cells = <1>;
+		pclkblock: pclkblock@4001e01c {
+			compatible = "renesas,ra-cgc-pclk-block";
+			reg = <0x4001e01c 4>, <0x40047000 4>, <0x40047004 4>,
+			      <0x40047008 4>;
+			reg-names = "MSTPA", "MSTPB","MSTPC",
+				    "MSTPD";
+			compatible = "renesas,ra-cgc-pclk-block";
+			#clock-cells = <0>;
+			sysclock-src = <RA_CLOCK_SOURCE_MOCO>;
+			status = "okay";
+
+			iclk: iclk {
+				compatible = "renesas,ra-cgc-pclk";
+				clk_div = <RA_SYS_CLOCK_DIV_16>;
+				#clock-cells = <2>;
+				status = "okay";
+			};
+
+			pclka: pclka {
+				compatible = "renesas,ra-cgc-pclk";
+				clk_div = <RA_SYS_CLOCK_DIV_16>;
```

Comment:
no underscores in properties please.

Reply:
I have created https://github.com/zephyrproject-rtos/zephyr/pull/78097.
I will fix it after this PR is merged.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/78097 is already merged.